README.md

```diff
@@ -44,23 +44,37 @@ For development purposes, you can set up `100eyes` using Docker. To perform the
     You can start all containers with a single command. `100eyes` can then be viewed in your browser at http://localhost:3000.
 
     ```bash
-    $ docker-compose up
+    $ docker compose up
     ```
 
-5. #### Seed database
+5. #### Create database
+
+   The database is not created in the steps above because it is a one-time task, whereas building and starting the services should be reproducible. Create the database with:
+
+   ```bash
+   $ docker exec 100eyes-app-1 rails db:create
+   ```
+
+6. #### Run the migrations
+
+   ```bash
+   $ docker exec 100eyes-app-1 rails db:migrate
+   ```
+
+7. #### Seed database
 
    One user has been created to use in development, since registration is disabled. You can create your own user in the `rails console` if you want, or you can run:
 
    ```bash
-   $ docker-compose exec app rails db:seed
+   $ docker exec 100eyes-app-1 rails db:seed
```

Comment:
the services have been renamed at some point in the past and the docs have not been updated, resulting in an error 
```bash
Error: No such container: app
```

Reply:
I think you might have forgotten `compose` in there. You can use `docker compose exec` to run a command inside a compose service, while you’d have to use the container name or id with `docker exec`.

So in this case the following should work just fine:

```
docker compose exec app rails db:seed
```

Reply:
Cool, I hadn't realized this referenced the service. I'll update it.